smart-contracts/contracts/OracleStorage.sol

```diff
@@ -18,27 +22,27 @@ contract OracleStorage {
     uint256 public consensusThreshold; // e.g. 75 = 75%
 
     /**
-    * @notice Tracks the number of OracleClaims made on an individual BridgeClaim
+    * @dev {DEPRECATED}
     */
-    mapping(uint256 => uint256) public oracleClaimValidators;
+    mapping(uint256 => uint256) private oracleClaimValidators;
 
     /**
-    * @notice mapping of prophecyid to validator address to boolean
+    * @dev {DEPRECATED}
     */
-    mapping(uint256 => mapping(address => bool)) public hasMadeClaim;
+    mapping(uint256 => mapping(address => bool)) private hasMadeClaim;
 
     /**
-    * @notice mapping of prophecyid to check if it has been redeemed
+    * @dev {DEPRECATED}
     */
-    mapping(uint256 => bool) public prophecyRedeemed;
+    mapping(uint256 => bool) private prophecyRedeemed;
```

Comment:
Is this used anywhere in the current or legacy code?

If not, let's remove.

Reply:
If this data structure is not in the current code off of develop, we can delete it altogether and will just need to readjust the storage padding as a result of this.